symfony/framework-bundle/3.3/src/Kernel.php

```diff
@@ -36,6 +36,8 @@ public function registerBundles()
 
     protected function configureContainer(ContainerBuilder $container, LoaderInterface $loader)
     {
+        // Feel free to remove the "container.autowiring.strict_mode" parameter
+        // if you are using Symfony 4.0+ as it's the default behavior
```

Comment:
maybe we should rather talk about `symfony/dependency-injection` than Symfony in general because that's the component that matters for this parameter

Reply:
Done thank you